docs(zh): translate other zh docs (#551)
web-infra-dev/rslib: website/docs/en/guide/solution/react.mdx

- **Type**: `string`
- **Default**: `'react'`

> When `runtime` is valued as `'automatic'`, you can specify the import path of the JSX transform through `importSource`.

Reviewer: ```suggestion
When `runtime` is set to `'automatic'`, you can specify the import path of the JSX transform through `importSource`.
```

Author: mark
